stock_orderpoint_other_availability/README.rst

```diff
@@ -0,0 +1,84 @@
+===================================
+Stock Orderpoint other availability
+===================================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fstock--logistics--warehouse-lightgray.png?logo=github
+    :target: https://github.com/OCA/stock-logistics-warehouse/tree/12.0/stock_orderpoint_other_availability
+    :alt: OCA/stock-logistics-warehouse
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/stock-logistics-warehouse-12-0/stock-logistics-warehouse-12-0-stock_orderpoint_other_availability
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/153/12.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+Allow to trigger orderpoints based also on the availability of other products.
```

Comment:
@SirTakobi Can you elaborate a little bit more the functional use case ?

As 'other' seems to me too vague.

Thanks

Reply:
Thanks for having a look!
I don't know exactly the functional use-case, I think it has something to do with MRP: you order the component when the final product is missing.
I have forwarded your inquiry to our functional team, they will hopefully give you a more detailed answer.

Reply:
> I don't know exactly the functional use-case, I think it has something to do with MRP: you order the component when the final product is missing.

It can't be linked to MRP as if products are well configured, orderpoints should behave correctly...

Reply:
@SirTakobi Any functional details about this ?

Reply:
@eLBati @mrcast please have a look